hack/create-bundle

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
```

Comment:
nit: other bash scripts here have `.sh` suffix so it should be here as well for consistency

Reply:
done